go-guru.el

```diff
@@ -231,9 +236,10 @@ output of the Go guru tool."
 	  (setq p (1- p)) ; exclude final space
 	  (let* ((posn (buffer-substring-no-properties start p))
 		 (flen (cl-search ":" posn)) ; length of filename
-		 (filename (if (< flen 19)
+		 (truncate (- (max go-guru-truncate-file-length 20) 1))
```

Comment:
I don't understand the use of `max` here. Can we not truncate to less than 20 characters?